docs/tutorials/connecting-to-the-internet-via-bluetooth.md

```diff
@@ -255,9 +255,54 @@ If you are using an older version, please upgrade.
 
         {% include /util/screenshot.html source="/images/brickman/networking-my-computer-0-disconnect-selected-online.png" %}
 
-*   {: tab="Windows"}__HELP WANTED!__ See [issue #287](https://github.com/ev3dev/ev3dev/issues/287).
+*   {: tab="Windows"}
+    These instructions were written using Windows 10 Enterprise.
+    It should work for other versions of Windows 10 as well.
+    {: .alert .alert-info}
+
+	1.	On the EV3, first verify that Bluetooth is powered on. In brickman,
+        open the *Wireless and Networks* menu and select *Bluetooth*. Make sure the
+        *Powered* checkbox is checked. The Bluetooth icon next to the battery in the
+        status bar also indicates that Bluetooth is powered on.
+		
+        {% include /util/screenshot.html source="/images/brickman/bluetooth-powered-visible-no-devices.png" %}
+	
+	2.	On your host computer, open the Action Center on the right end of the Taskbar and right-click on the Bluetooth Button.
```

Comment:
![bluethoothxname](https://user-images.githubusercontent.com/30458827/28706393-63a1e274-7374-11e7-8dea-48e32951642e.PNG)
I would like to implement this Picture too